core/block_svg.ts

```diff
@@ -888,18 +886,6 @@ export class BlockSvg
     }
   }
 
-  /**
-   * Get the comment icon attached to this block, or null if the block has no
-   * comment.
-   *
-   * @returns The comment icon attached to this block, or null.
-   * @deprecated Use getIcon. To be remove in v11.
-   */
-  getCommentIcon(): CommentIcon | null {
```

Comment:
Could you revert the changes to this file? We're not ready to delete this yet. We're deprecating this in v10 (coming out at the end of this month) and then deleting it in v11!

Reply:
on it, thank you!